setup.py

```diff
@@ -76,6 +76,7 @@
         ],
         'pyls': [
             'autopep8 = pyls.plugins.autopep8_format',
+            'flake8 = pyls.plugins.flake8_lint',
```

Comment:
possible to add a test for this functionality?

Reply:
done